CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# dbt_quickbooks v0.16.0
+[PR #143](https://github.com/fivetran/dbt_quickbooks/pull/143) introduces the following updates:
+
+## Upstream Source Package Updates
+- Deleted records have been previously been brought into the `dbt_quickbooks` package. To ensure accuracy in reporting, the following updates were made in the [v0.11.0 release](https://github.com/fivetran/dbt_quickbooks_source/releases/tag/v0.11.0) of `dbt_quickbooks_source`:
+- We introduced the `_fivetran_deleted` field to filter out deleted records from the following staging models:
+  - `stg_quickbooks__account`
+  - `stg_quickbooks__bundle`
+  - `stg_quickbooks__customer`
+  - `stg_quickbooks__department`
+  - `stg_quickbooks__item`
+  - `stg_quickbooks__vendor`
```

Comment:
Can you add which end models could potentially be affected by this change? This would help customers check their data if they need to.

Reply:
It's impacting all models haha, so just tweaked line 13.